data/scripts/talkactions/commands/attributes.lua

```diff
@@ -0,0 +1,322 @@
+local talk = TalkAction("/attr")
+
+local itemFunctions = {
+    ["actionid"] = {
+        isActive = true,
+        targetFunction = function(item, target)
+            return item:setActionId(target)
+        end
+    },
+
+    ["uniqueid"] = {
+        isActive = true,
+        targetFunction = function(item, target)
+            return item:setAttribute(ITEM_ATTRIBUTE_UNIQUEID, target)
+        end
+    },
+
+    ["description"] = {
+        isActive = true,
+        targetFunction = function(item, target)
+            return item:setAttribute(ITEM_ATTRIBUTE_DESCRIPTION, target)
+        end
+    },
+
+    ["name"] = {
+        isActive = true,
+        targetFunction = function(item, target)
+            return item:setAttribute(ITEM_ATTRIBUTE_NAME, target)
+        end
+    },
+
+    ["remove"] = {
```

Comment:
We already have `/r` and it works better because we can set the amount we want to erase, also everyone already knows this command and I doubt they want to use this method instead